HAR-7075 Materiaaleille yksiköt

Joonas-M-S opened the pull request.

Joonas-M-S committed:
Lisää materiaalit näkymään frontille ja materiaalit taulukkoon yksikkö sarake

Joonas-M-S committed:
Materiaalin tallennus yksikköineen onnistuu

Joonas-M-S committed:
Lisää yksikkösarake häiriötilanteen varaosattaulukkoon. Lisää yksikko vv_materiaalilistaus taulukkoon

Joonas-M-S committed:
Mahdollista materiaalien kirjaus materiaalinäkymästä

Joonas-M-S committed:
Lisää kaikille olemassaoleville matskuille yksiköksi 'kpl'

Joonas-M-S committed:
refaktoroi hinnoittelu headerit kanavien muutos- ja lisätöissä. Muuta hinnoittelun kantakonstraintti takaisin alkuperäiseen. Lisää testidataan yksiköt.

Joonas-M-S committed:
Lisää materiaalin yksikkö hinnoitteluun

Joonas-M-S committed:
Korjaa testejä ja näytä materiaalin yksikkö häiriötilanne lomakkeella

Joonas-M-S committed:
Lisää yksikkö toimenpiteisiin ja refaktoroi hieman

Joonas-M-S committed:
Merge branch 'develop' into HAR-7075

Joonas-M-S committed:
Näytä käytetyn materiaalin yksikkö hinnoitteluleijukkeella

Joonas-M-S committed:
Merge branch 'develop' into HAR-7075

Joonas-M-S committed:
Korjaa kanavatoimenpiteet testi

Joonas-M-S committed:
Korjaa materiaalit testit

Joonas-M-S committed:
Korjaa kokonaishintaiset frontti testit

Joonas-M-S committed:
Merge branch 'develop' into HAR-7075

Joonas-M-S committed:
Korjaillaan fronttitestejä

Joonas-M-S committed:
mahdollista materiaalin yksikön muokkaaminen

Joonas-M-S committed:
Yksikön tallentaminen hinnoittelussa

Joonas-M-S committed:
Korjaa pari testiä ja lisää assertteja

Joonas-M-S committed:
Lisää muokkaustiedot

Kauko commented on src/cljc/harja/domain/vesivaylat/materiaali.cljc:
```diff
@@ -17,13 +17,13 @@
 (s/def ::poista-materiaalikirjaus (s/keys :req [::id ::urakka-id]))
-(s/def ::muuta-materiaalien-alkuperainen-maara (s/keys :req [::urakka-id]
-                                                       :req-un [::uudet-alkuperaiset-maarat]))
+(s/def ::muuta-materiaalien-alkuperainen-maara-ja-yksikko (s/keys :req [::urakka-id]
```
Laittaisin näihin perään -kysely, koska on kyseessä kysely-spec

Joonas-M-S committed:
Rename V1_687__.sql to V1_688__.sql

Joonas-M-S committed:
Merge branch 'develop' into HAR-7075

Joonas-M-S committed:
Lisää yksikkö myös vetolaatikon listauksille.

Joonas-M-S committed:
Korjaa vv_materiaali testidata

Joonas-M-S committed:
Mahdollista hälytysrajan muokkaaminen

Joonas-M-S committed:
Rename V1_688__.sql to V1_689__.sql

Joonas-M-S committed:
Merge branch 'develop' into HAR-7075

Joonas-M-S committed:
Korjaa materiaalitesti

Joonas-M-S committed:
Lisää testi hälytysrajalle

Joonas-M-S committed:
Korjaa materiaalien sorttaus ja alkuperäinen määrä ongelmat

Joonas-M-S committed:
Mutta toimenpiteellä lisätyn materiaalikirjauksen viestiä

Joonas-M-S committed:
Merge branch 'develop' into HAR-7075

Joonas-M-S committed:
Korjaa testi

Kauko committed:
Merge branch 'develop' into HAR-7075

Joonas-M-S committed:
Testien korjailua

Joonas-M-S committed:
Merge branch 'HAR-7075' of https://github.com/finnishtransportagency/harja into HAR-7075

Joonas-M-S committed:
Muuta lisätieto viestejä

Joonas-M-S committed:
Merge branch 'develop' into HAR-7075

Joonas-M-S committed:
Muuta materiaalin merkkausaikaa

Kauko committed:
Anna uudelle materiaalikirjaukselle toimenpiteen pvm

Kauko committed:
Merkkaa materiaalikirjaus muokatuksi, kun toimenpiteen pvm muuttuu

Kauko committed:
Tallenna häiriötilanteen materiaalikirjaukselle pvm

Kauko committed:
Merkkaa muokatuiksi materiaalikirjaukset, joiden pvm on eri kuin häiriötilanteella

Kauko committed:
Siirrä häiriötilanteet-välilehteä oikealle

Oletuksena laadunseurantanäkymässä on valittuna tarkastukset-välilehti.
Näyttää hassulta, että valittuna on välilehti, joka ei ole ensimmäinen
vasemmalta katsottuna.

Kauko committed:
Merge remote-tracking branch 'origin/develop' into HAR-7075

Kauko committed:
Merge branch 'develop' into HAR-7075

Kauko committed:
Palauta poistettu newline
